frontend/src/lib/lemon-ui/LemonCalendar/LemonCalendar.scss

```diff
@@ -7,6 +7,8 @@
     // Tricky: needs to match the equivalent height button from LemonButton.scss
     --lemon-calendar-time-button-height: 2.3125rem;
 
+    height: 305px;
```

Comment:
Might be worth making this a variable or at least saying what this value is composed of

Reply:
Added! good call on the self documentation.